go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 	gopkg.in/yaml.v3 v3.0.1
 )
+
+replace github.com/canonical/x-go => github.com/rebornplusplus/canonical-x-go v0.0.0-20230309051359-1ec2b2ee698e
```

Comment:
Note the use of Rafid's fork+branch here -- we'll need to merge the `shlex.Quote` and `shlex.Join` PR (https://github.com/canonical/x-go/pull/19) to avoid this.

Reply:
@rebornplusplus Does this need to be updated to 61c9f268b5f0?

Reply:
The `shlex.Quote` and `shlex.Join` PR (https://github.com/canonical/x-go/pull/19) PR has been merged recently.
I updated the module version in https://github.com/canonical/pebble/pull/191/commits/43ef4ef1b60f77561f8da9b016fdf81f412f34b3.